### DISCUSSION TITLE
Idea for a report system

### DISCUSSION THREAD

**Discussion by XxDuckbusterxX** [Zrips/CMI#1663] [Feature Requests]
Hi,

first I wanted to say that the CMI is a great plugin.
What I miss or have not found yet is a report system.
Players can report players with justifications. The report message is stored in a list or database and can be viewed by an admin and teleported directly to the scene via the message.

Would be cool if such a function might be installed.

**Comment by mercurialmusic** [comment]
Or a ticket system in general. I've bounced around a couple ticket reporting plugins over the years as they died out. That would be yet another plugin that CMI makes redundant. :)
